packages/build/tests/core/tests.js

```diff
@@ -536,23 +527,3 @@ test('Generates a `manifest.json` file when running outside of buildbot', async
   t.is(typeof timestamp, 'number')
   t.is(manifestVersion, 1)
 })
-
-test('Generates a `manifest.json` file when the `buildbot_create_functions_manifest` feature flag is set', async (t) => {
```

Comment:
Why are we removing this test? We still want to ensure that the manifest is created, we just don't need to pass the `buildbot_create_functions_manifest` feature flag anymore.

Reply:
Addressed